tests/test_ops.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import torch_ttnn
+import unittest
+from torch_ttnn import ttnn
+import tt_lib
+
+from torch_ttnn.utils import check_with_pcc
+
+class TransposeModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, dim0, dim1):
+        return torch.transpose(x, dim0, dim1)
+
+    def input_shapes(self):
+        return [(32, 64, 128), 0, 2]
+
+
+class TestModules(unittest.TestCase):
+    def setUp(self):
+        # Open device 0
+        self.device: ttnn.Device = ttnn.open_device(device_id=0)
+        # For AutoFormat
+        tt_lib.device.SetDefaultDevice(self.device)
+
+    def tearDown(self):
+        # Close the device
+        ttnn.close_device(self.device)
+
+    def test_transpose(self):
+        m = TransposeModule()
+        input_shapes = m.input_shapes()
+        input = torch.randint(0, 3, input_shapes[0]).type(torch.bfloat16)
+        dim0 = input_shapes[1]
+        dim1 = input_shapes[2]
+        result_before = m.forward(input, dim0, dim1)
+        option = torch_ttnn.TorchTtnnOption(device=self.device)
+        option.gen_graphviz = True
+        # The compilation is lazy, so we need to run forward once to trigger the compilation
+        m = torch.compile(m, backend=torch_ttnn.backend, options=option)
+        result_after = m.forward(input, dim0, dim1)
+        option._out_fx_graphs[0].print_tabular()
```

Comment:
do we need this print?

Reply:
This printing is there for all existing tests, so I included it.

Reply:
Yeah all the tests have this from the start. I think it should be fine since it only shows up if we ask pytest to be verbose.